### ISSUE TITLE
Camera and cropping view on AVD 35-ext13 

### ISSUE THREAD

**Issue by flipflop97** [ivpusic/react-native-image-crop-picker#2102]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.41.4
- react-native v0.75.4

### Platform
Tell us to which platform this issue is related

- [ ] iOS
- [x] Android

### Expected behaviour

Camera picker should have device insets to make sure elements are not drawn underneath the system bars

### Actual behaviour

Camera picker ad cropper draw under system bars in some circumstances

### Steps to reproduce

1. Setup project which targets Android 15, and uses the camera image crop picker

2. Run on Android virtual device "Pixel 8 Pro - API 35-ext13"

3. Open the camera picker and cropper, and notice the system bars cover the top and bottom

*Note that I cannot reproduce this on a real Pixel 8 Pro (Android 14), this has correct insets with edge-to-edge mode enabled*

### Attachments

<img width="489" alt="image" src="https://github.com/user-attachments/assets/cf405b47-f8f9-455d-ad00-d05a0510f467">

**Comment by flipflop97** [comment]
Update: Just tested this on my real Pixel 8 Pro with Android 15, and the issue also appears there

**Comment by SectionTN** [comment]
having the same issue

**Comment by mozzius** [comment]
Also getting this. It's due to this:

https://developer.android.com/about/versions/15/behavior-changes-15#window-insets

**Comment by almastabssam** [comment]
@ivpusic @mozzius Please update library with above changes and create the PR. Thanks

**Comment by mozzius** [comment]
Here's the patch file I made for Bluesky: https://github.com/bluesky-social/social-app/blob/main/patches/react-native-image-crop-picker+0.41.6.patch

Feel free to make a PR, but it is a temporary solution. It would be better to handle the insets properly

**Comment by almastabssam** [comment]
thanks, i already made a patch. Please add it in library. thanks
Almas Tabssam
Senior React Native Developer
Mobile No: +923137833381

**Comment by mozzius** [comment]
I'd love to but I am not a maintainer and I don't know Android dev :P

**Comment by rpopovici** [comment]
This happens only because you set targetSdkVersion=35. You can fix it by setting targetSdkVersion=34 or by settings compileSdkVersion=35 and setting `<item name="android:windowOptOutEdgeToEdgeEnforcement">true</item>` in AndroidManifest.xml

 https://www.droidcon.com/2024/09/20/%F0%9F%9A%A8-google-secretly-adds-opt-out-api-for-android-15s-edge-to-edge-behaviour-%F0%9F%9A%A8/

**Comment by designervenkat** [comment]
> This happens only because you set targetSdkVersion=35. You can fix it by setting targetSdkVersion=34 or by settings compileSdkVersion=35 and setting `<item name="android:windowOptOutEdgeToEdgeEnforcement">true</item>` in AndroidManifest.xml
> 
> https://www.droidcon.com/2024/09/20/%F0%9F%9A%A8-google-secretly-adds-opt-out-api-for-android-15s-edge-to-edge-behaviour-%F0%9F%9A%A8/

This solutons works for me ! Thanks @rpopovici

**Comment by joaolfr** [comment]
Having the same issue

**Comment by flipflop97** [comment]
Adding `<item name="android:windowOptOutEdgeToEdgeEnforcement">true</item>` would since this will likely be removed in the nearby future and targeting SDK 35 will be forced, it needs a proper solution.

**Comment by danger9224** [comment]
Yeah, Android 16 DP1 already removed the option to opt-out of the edge-to-edge enforcement: https://android-developers.googleblog.com/2025/02/second-beta-android16.html

**Comment by mozzius** [comment]
Update: this library is totally dead and unmaintained (800 open issues!!), so we made our own one from scratch: https://github.com/bluesky-social/expo-image-crop-tool

**Comment by tapz** [comment]
@mozzius This seems to require integrating expo to the RN app. Tried that and made too many changes to my app, so reverted.

**Comment by AnthonyJamez12** [comment]
Here’s what worked for me:

**In `AndroidManifest.xml`:**

```xml
<activity
    android:name="com.yalantis.ucrop.UCropActivity"
    android:screenOrientation="nosensor"
    android:windowLayoutInDisplayCutoutMode="never"
    android:theme="@style/Ecift.CropTheme"
    tools:replace="android:theme" />
```

**In `res/values/styles.xml`:**

```xml
<style name="Ecift.CropTheme" parent="Theme.AppCompat.Light.NoActionBar">
    <item name="colorPrimary">#000000</item>
    <item name="colorAccent">#FF4081</item>
    <item name="android:windowOptOutEdgeToEdgeEnforcement">true</item>
</style>
```

These changes fixed the issue for me.
Hope this helps someone else!

**Comment by bhanuka-viraj** [comment]
I've got the same issue. Is there any permanent fix ?

**Comment by MaioranoB** [comment]
Still relevant

**Comment by hamzamahmood-aai** [comment]
Just add StatusBar and SafeAreaView on screen on which you use picker. this will resolve this issue no need for patch.
`
<StatusBar barStyle="light-content" backgroundColor={colors.black} />
      <SafeAreaView style={[profileStyles.container, { backgroundColor: colors.black }]}>
        {/* Your screen content here */}
      </SafeAreaView>
`

**Comment by renishmithani** [comment]
+1

**Comment by Flyview16** [comment]
### This worked for me (I'm using v.11)

####  Opt out of edge-to-edge in styles

Create or update this file:
`android/app/src/main/res/values-v35/styles.xml`

```xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
  <style name="Ucrop.CropTheme" parent="Theme.AppCompat.Light.NoActionBar">
    <item name="android:windowOptOutEdgeToEdgeEnforcement">true</item>
  </style>
</resources>
```

#### Then reference that theme in your manifest
```xml
<activity
  android:name="com.yalantis.ucrop.UCropActivity"
  android:screenOrientation="portrait"
  android:theme="@style/Ucrop.CropTheme" />
```

#### Add the new light-mode flags in your codes
```dart
AndroidUiSettings(
 ...
  // keep bars visible and icons bright
  statusBarLight: false,
  navBarLight: false,
)
```

**Comment by Abusayid693** [comment]
Any proper solution for this
